README.md

```diff
@@ -18,48 +18,40 @@ $ <newline_separated_list_of_repositories> | ./bin/ebi.js <command>
 
 ### Examples
 
+Prerequisites
+
+-   See [GitHub personal access token security](#github-personal-access-token-security) to set up the GitHub token
+
 Show help
 
     ./bin/ebi.js --help
 
 Determine whether a repo has a `Procfile`
 
 ```
-$ echo -e "Financial-Times/next-search-page\nFinancial-Times/next-gdpr-tests" | ./bin/ebi.js --token $GITHUB_PERSONAL_ACCESS_TOKEN contents Procfile
+$ echo -e "Financial-Times/next-search-page\nFinancial-Times/next-gdpr-tests" | ./bin/ebi.js contents Procfile
 Financial-Times/next-search-page
 404 ERROR: file 'Procfile' not found in 'Financial-Times/next-gdpr-tests'
 ```
 
 Find all the `node` engines and their versions in `package.json`
 
 ```
-echo -e "Financial-Times/next-search-page\nFinancial-Times/next-gdpr-tests" | ./bin/ebi.js --token $GITHUB_PERSONAL_ACCESS_TOKEN package:engines
+echo -e "Financial-Times/next-search-page\nFinancial-Times/next-gdpr-tests" | ./bin/ebi.js package:engines
 Financial-Times/next-search-page	node@8.15.0
 Financial-Times/next-gdpr-tests	node@^8.9.4	npm@5.7.1
 ```
 
 ## GitHub personal access token security
 
-This tool requires a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
-with all `repo` scopes. This is _very powerful_ as it has access to modify a
-repository's settings, so it is strongly recommended that you store this token
-securely.
-
-Once you have created a [new GitHub personal access token with all `repo` scopes](https://github.com/settings/tokens/new?description=Manage%20GitHub%20Apps%20CLI&scopes=repo 'Click here to create a new GitHub personal access token'),
-you can store it in an environment variable and pass it to `manage-github-apps`
-whenever you run a command that requires the `--token` option:
-
-```sh
---token $GITHUB_PERSONAL_ACCESS_TOKEN
-```
+This tool requires a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with all `repo` scopes. This is _very powerful_ as it has access to modify a repository's settings, so it is strongly recommended that you store this token securely.
 
-You should avoid passing your GitHub personal access token directly to any CLI
-arguments as then it will be visible in your shell history.
+To store the token securely:
 
```

Comment:
Can we link this wiki page instead? - https://github.com/Financial-Times/next/wiki/How-to-store-and-access-a-GitHub-personal-access-token-securely. I think it's clearer than the instructions here. We can always specify that a fallback is to use `token=xxxx` for those who don't have access to our wiki

Reply:
Yah that makes more sense - avoids us having to update info in two places!